raddb/sites-available/coa-relay

```diff
@@ -0,0 +1,289 @@
+			foreach &control:Tmp-String-0 {
+				if ( "%{Foreach-Variable-0}" =~ /([^#]*)#(.*)/ ) {
+					update request {
+
+						COA-Packet-DST-IP-Address := "%{1}"
+						COA-Acct-Session-Id := "%{2}"
+
+						# Set the CoA/Disconnect port
+						COA-Packet-DST-Port := 1700
+
+						# NASs don't usually want to
+						# see User-Name, and
+						# SQL-User-Name is left over
+						# from the xlat
+						#
+						User-Name !* ANY
+						SQL-User-Name !* ANY
+
+					}
+					if (COA-Packet-Type != "" && \
+					    COA-Packet-DST-IP-Address && \
+					    COA-Acct-Session-Id != "") {
+
+						detail_coa.accounting
+
+					}
+				}
+			}
+		} else {
+			# No sessions found
+			reject
+		}
+
+	}
+}
+
+#
+#  Detail file reader that processes the queue of CoA/Disconnect requests
+#
+server coa-buffered-reader {
+	listen {
+		#
+		#  See sites-available/buffered-sql for more details on
+		#  all the options available for the detail reader.
+		#
+		type = detail
+		filename = "${radacctdir}/detail_coa"
+		load_factor = 90
+		track = yes
+	}
+
+	accounting {
+
+		switch &COA-Packet-Type {
+
+			case "Disconnect-Request" {
+				update {
+					disconnect:Packet-DST-IP-Address = \
+						&COA-Packet-DST-IP-Address
+					disconnect:Packet-DST-Port = \
+						&COA-Packet-DST-Port
+					disconnect:Acct-Session-Id = \
+						&COA-Acct-Session-Id
+					disconnect:Event-Timestamp = "%l"
+					disconnect:Message-Authenticator = 0x00
+				}
+			}
+
+			case "CoA-Request" {
+				update {
+
+					#  Include given attributes
+					coa: += request:[*]
+
+					coa:Packet-DST-IP-Address = \
+						&COA-Packet-DST-IP-Address
+					coa:Packet-DST-Port = \
+						&COA-Packet-DST-Port
+					coa:Acct-Session-Id = \
+						&COA-Acct-Session-Id
+					coa:Event-Timestamp = "%l"
+					coa:Message-Authenticator = 0x00
+
+				}
+			}
+
+		}
+
+		ok
+	}
+}
+
+
+#  The CoA packet is in the "proxy-request" attribute list.
+#  The CoA reply (if any) is in the "proxy-reply" attribute list.
+#
+server originate-coa {
+	pre-proxy {
+		update proxy-request {
+
+			#  Specify the source IP address for the request
+			NAS-IP-Address = 10.1.2.3
+
+		}
+	}
+
+	#
+	# Handle the responses here.
+	#
+	post-proxy {
+		switch &proxy-reply:Packet-Type {
+			case CoA-ACK {
+				ok
+			}
+
+			case CoA-NAK {
+				# the NAS didn't like the CoA request
+				ok
+			}
+
+			case Disconnect-ACK {
+				ok
+			}
+
+			case Disconnect-NAK {
+				# the NAS didn't like the Disconnect request
+				ok
+			}
+
+			# Invalid packet type.  This shouldn't happen.
+			case {
+				fail
+			}
+		}
+
+		#
+		#  These methods are run when there is NO response
+		#  to the request.
+		#
+		Post-Proxy-Type Fail-CoA {
+			ok
+		}
+
+		Post-Proxy-Type Fail-Disconnect {
+			ok
+		}
+	}
+}
+
+
+#
+#  Homeserver CoA / Disconnect endpoints
+#
+#  See proxy.conf for more details on configuring a home_server and
+#  home_server_pool.
+#
+home_server coa-nas1 {
+	type = coa
+	ipaddr = 10.0.0.1
```

Comment:
This should be `192.0.2.1`, which is the canonical "documentation IP network".  Along with a comment "please edit this..."